### PR TITLE
Correct minor surface errors

### PR DESCRIPTION
**Pull request by O-I**
Just a few fixes while I was reading this. A good guide to bad data. Very useful resource.

Cheers,
O-I

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -216,7 +216,7 @@ The maximum number of rows an old-fashioned Excel spreadsheet was allowed to hav
 
 For reasons beyond obscure, Excel's default date from which it counts all other dates is `January 1st, 1900`, *unless* you're using Excel on a Mac, in which case it's `January 1st, 1904`. There are a variety of ways in which data in Excel can be entered or calculated incorrectly and end up as one of these two dates. If you spot them in your data, it's probably an issue.
 
-Many databases and applications will often generate a date of `1970-01-01T00:00:00Z` or `1969-12-31T24:59:59Z` which is the [Unix epoch for timestamps](https://en.wikipedia.org/wiki/Unix_time#Encoding_time_as_a_number). In other words this is what happens when a system tries to display an empty value or a `0` value as a date.
```
Comment (igorbrigadir) [inline README.md]: Why would it be T24:59:59Z and not T23:59:59Z?

Comment (bryant1410) [inline README.md]: The change is the other way around. He's leaving 23 instead of 24.

Comment (O-I) [inline README.md]: That is correct, I changed it from 24 to 23.

Comment (igorbrigadir) [inline README.md]: Ha! I got it backwards! My mistake - Thanks!

**Comment by onyxfish** [comment]
Thanks!

**Pull request merged**